src/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
```

Comment:
- Please, kindly remove the linter disabled line a fix it by writing code that conforms to the linter to help our codebase be maintainable.